### DISCUSSION TITLE
explain this crud code

### DISCUSSION THREAD

**Discussion by macMikey** [serverpod/serverpod#2815] [Q&A] [answered]
i don't know if i just don't understand, or if these are mistakes, and i should fix them an issue a PR:
on the [crud documentation page, in "Update a specific column"](https://docs.serverpod.dev/concepts/database/crud#update-a-specific-column), there are two examples, that don't look right to me. they have the same issue:
```dart
var company = await Company.db.findById(session, companyId); 
company.name = 'New name';
company.address = 'Baker street'
var updatedCompany = await Company.db.updateRow(session, company, columns: [Company.t.name]);
```

in the last line, there is `columns: [Company.t.name]);`. is it me? that **.t** looks like a mistake.

**Comment by hampuslavin** [comment, marked as answer]
Hi @macMikey! 👋 
Thanks for being observant, this is indeed a typo! The `columns` parameter should be a function returning a list (i.e. `columns: (t) => [t.name]`, and just FYI this would be equivalent: `columns: (_) => [Company.t.name]`) If you want you are more than welcome to fix this error by opening a PR on the docs repo: https://github.com/serverpod/serverpod_docs 🌟

**Comment by macMikey** [comment]
https://github.com/serverpod/serverpod_docs/pull/171
